### ISSUE TITLE
Accessing the Logits

### ISSUE THREAD

**Issue by AlexanderZeilmann** [MIC-DKFZ/nnInteractive#12]
Is it possible to access the segmentation logits, i.e., the raw output of the neural network before the final argmax operation?

**Comment by AlexanderZeilmann** [comment]
The reason why I am asking is that this would make it easier to
1. estimate the uncertainty of the segmentation at each pixel and
2. support multi-class segmentation.

From what I see here :
https://github.com/MIC-DKFZ/nnInteractive/blob/9192ecf75f2302c4efc7d7657a083c8e967b2ad8/nnInteractive/inference/inference_session.py#L443
and here:
https://github.com/MIC-DKFZ/nnInteractive/blob/9192ecf75f2302c4efc7d7657a083c8e967b2ad8/nnInteractive/inference/inference_session.py#L566

it looks like as if this is currently not possible.

Maybe adding something like

```py
logits = self.network(input_for_predict[None])[0]
if write_logits:
   target_buffer_logits = logits
pred = logits.argmax(0).detach() 
```

would make sense to support the above mentioned use cases.
